pymchelper/writers/mcsum.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import time
+import urllib.parse
+
+logger = logging.getLogger(__name__)
+
+
+class MCSumWriter:
```

Comment:
Do we really need a new writer for that functionality ?
Could it be part of `inspect` command ?
Like:
`convertmc inspect blabla.bdo --summary`
or
`convertmc inspect --many "*.bdo" --summary`
?

Reply:
I thought of additional options for what format you would like to have. Also it may collide with inspect -d option, and may get more complex if also the scoring field will be filled out.

Other questions: we could hardcode also SH references, for convenience and make reference list at bottom.